DOCS-9255-mule-domain-limitation (#71)
mulesoft/docs-visualizer: modules/ROOT/pages/_partials/include-mule-domain-support.adoc

> Apps and APIs that are associated with a Mule domain appear in the Anypoint Visualizer application network, but metrics and connection information related to these apps and APIs do not appear.

Author: ```suggestion
Apps and APIs that are associated with a Mule domain appear in the Anypoint Visualizer application network, but metrics and connection information related to these apps and APIs may be inaccurate.
```

> Apps and APIs that are associated with a Mule domain appear in the Anypoint Visualizer application network, but metrics and connection information related to these apps and APIs may be inaccurate.

Reviewer: Per the style guide, I think using **might** instead of **may** fits better here.

Author: ```suggestion
Apps and APIs that are associated with a Mule domain appear in the Anypoint Visualizer application network, but metrics and connection information related to these apps and APIs might be inaccurate.
```
